freqtrade/exchange/bittrex.py

```diff
@@ -23,3 +24,28 @@ class Bittrex(Exchange):
         },
         "l2_limit_range": [1, 25, 500],
     }
+
+    def setup_leveraged_enter(
+        self,
+        pair: str,
+        leverage: float,
+        amount: float,
+        quote_currency: Optional[str],
+        is_short: Optional[bool]
+    ):
+        raise OperationalException("Bittrex does not support leveraged trading")
```

Comment:
You should implement these methods in the parent exchange class - otherwise you'll have to copy/paste the same message to all potential classes (without the guarantee that you'll cover all future exchange classes (or exchanges that don't have their own subclass).

Reply:
There is one in the main exchange class that says it's not supported using freqtrade. This one is more specific, saying that the exchange doesn't support it at all

Reply:
There's little point doing it THIS way (you should never even reach this point in the first place) - as it would only error out once you try to open a trade - this must however be flagged at startup as "invalid configuration".


We'll need to have a flag (most likely in `_ft_has`) which "enables" an exchange for "optional" features/modes - similar to `stoploss_on_exchange` (in `_ft_has`) - but most likely more complex as this will need to cover multiple cases (leverage (for longs), short, ... ).

Obviously, the same checks that we use for `stoploss_on_exchange` (abort startup if an invalid configuration for the selected exchange is detected) will also need to be added here.